### DISCUSSION TITLE
how about  changing DEFAULT_TCP_BLKSIZE?

### DISCUSSION THREAD

**Discussion by rameble** [esnet/iperf#1384] [Q&A] [answered]
I want to modify DEFAULT_TCP_BLKSIZE from (128 * 1024) to (10 * 1024) because of limited storage.   I don't know whether this modification may lead to some side effect or incorrect test result. Does anyone know? 

Thanks a lot.

**Comment by davidBar-On** [comment, marked as answer]
Instead of changing `DEFAULT_TCP_BLKSIZE` to 10KB you can just set the `-l` option when running the client, i.e `-l 10K`.  This has the equivalent effect, but without the need to rebuild iperf3 (and you can change the length for each test).

The effect of reducing the block size is usually reduced maximum throughput.

**Comment by rameble** [comment]
thanks for your reponse.
